modelon/impact/client/entities/workspace.py

```diff
@@ -132,6 +193,26 @@ def upload_model_library(self, path_to_lib: str):
             workspace.upload_model_library('C:/A.mo')
             workspace.upload_model_library('C:/B.mol')
         """
+        logger.warning(
+            "The 'upload_model_library' is deprecated and will be removed in a future"
+            " release. Use 'upload_dependency' method instead!"
+        )
+        self._workspace_sal.library_import(self._workspace_id, path_to_lib)
```

Comment:
It will probably be more sound to just remove this method and end-point because it will not behave as you expect. Better to get users to use the new stuff. So we should have clean commit with BREAKING CHANGE and that removes this method, then we get correct versioning and users can check for breaking changes in the release notes.